change/@fluentui-react-carousel-preview-5ce398b2-e93c-475a-adb6-4c4e71dd7011.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32422&commitId=2d8dbcef2fdd77067789ae79380c41f75b41880a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|
|Avatar Converged.badgeMask.chromium.png|55| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|
|Drawer.overlay drawer full.chromium.png|1139| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>